app/src/main/java/org/simple/clinic/summary/PatientSummaryProfile.kt

```diff
@@ -14,4 +14,8 @@ data class PatientSummaryProfile(
     val phoneNumber: PatientPhoneNumber?,
```

Comment:
Can you change the commit message to fit in one line, please?

Reply:
Done.